src/github/utils/plugins.ts

```diff
@@ -45,7 +45,7 @@ async function fetchWorkerManifest(url: string): Promise<Manifest | null> {
   if (_manifestCache[url]) {
     return _manifestCache[url];
   }
-  const manifestUrl = `${url}/manifest.json`;
+  const manifestUrl = `${url}/manifest`;
```

Comment:
No file extension? That seems wrong. Even if you changed it on the worker, it should have the full file name.

Reply:
If a `GET` done to an endpoint, it is strange to have the extension in the URL, which is why I removed it. This endpoint doesn't download a file but gives you a JSON response body.

Reply:
The file is a json file. It should be with the file extension for sure. Same as they do with chrome extensions.

Reply:
But it's an API endpoint, if it was a file we would return a blob with a binary that would downloaded. I changed it because eventually we could do `POST` to update or do checks, which was the first planned implementation.